packages/twenty-front/src/modules/settings/data-model/constants/SettingsFieldCurrencyCodes.ts

```diff
@@ -55,7 +93,7 @@ export const SETTINGS_FIELD_CURRENCY_CODES: Record<
   },
   HKD: {
     label: 'Hong Kong dollar',
-    Icon: IconCurrencyDollar,
+    Icon: IconCurrencyRenminbi,
```

Comment:
is this correct?

Reply:
According to Wikipedia, these are interchangeable, so I went with Renminbi to highlight the difference. 
![image](https://github.com/user-attachments/assets/47aafad4-a302-4f8a-b53b-c257d9045897)
Sources: https://en.wikipedia.org/wiki/Currency_symbol https://en.wikipedia.org/wiki/Hong_Kong_dollar

Reply:
Ok!